docs: simplify the react quick start (#1652)
alchemyplatform/aa-sdk: docs/pages/react/quickstart/new-project.mdx

---

> title: Quickstart (New Project)

Reviewer: ```suggestion
title: Quickstart
```

description: Get started in 5 minutes with Alchemy Smart Wallets using our NextJS template with embedded smart wallets and social login.

> text: Quickstart (New Project)

Reviewer: ```suggestion
text: Quickstart
```

link: /react/quickstart/new-project
slug: wallets/react/quickstart/new-project
---

> **By the end of this tutorial, you'll have a working NextJS application with embedded smart wallets and social login, complete with gas sponsorship enabled out of the box.**

Reviewer: ```suggestion
**By the end of this tutorial, you'll have a working NextJS application with embedded smart wallets, social login, and gasless transactions.**
```

Reviewer: We should probably add a link to the template github repo directly here so they can go check it out if needed

The fastest way to get started is using our **Quickstart widget** in the Alchemy Dashboard.

## 1. Use the Dashboard Quickstart

1. Go to the Smart Wallets Quickstart in the [Alchemy Dashboard](https://dashboard.alchemy.com/services/smart-wallets/quickstart)
2. Accept the terms and conditions if you haven't already

> 3. Follow the 3-step process:

Reviewer: I'm thinking we should also give the quickstart snippets here so it's easier to get.

   - **Create new app**: Copy the provided `npx create-next-app` command
   - **Set up environment**: Copy the pre-configured `.env` variables (includes your API key and policy ID)
   - **Run app**: Use the provided run command

<img
  src="/images/wallets/dashboard-quickstart-widget.png"
  alt="Alchemy Dashboard quickstart widget showing the 3-step process"
  style={{ width: "100%", maxWidth: "800px" }}
/>

The quickstart widget automatically provides you with:

> - ✅ Pre-configured API key (`NEXT_PUBLIC_ALCHEMY_API_KEY`)

Author: Yeah I'm open to adding more context about the api keys.  I think from the divo definition of a tutorial, the tutorial should just work and users shouldn't need info that isn't relevant to running said tutorial.

I was trying to intentionally keep this as light as possible.  I can see that it's still valuable to add some info about the api keys though
